<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditScientist</class>
 <widget class="QDialog" name="EditScientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Edit Scientist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="editScientistNameError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editScientistName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Sex:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="editScientistSex"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="labelyob">
       <property name="text">
        <string>Year Of Birth:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="editScientistYearOfBirthError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editScientistYearOfBirth"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Year died (optional):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="editScientistYearOfDeathError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editScientistYearOfDeath"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="buttonEditScientist">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancelEditScientist">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>scientistYearDied</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>scientistName</zorder>
  <zorder>scientistSex</zorder>
  <zorder>label_10</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>scientistYearBorn</zorder>
  <zorder>label</zorder>
  <zorder>label</zorder>
  <zorder>editScientistName</zorder>
  <zorder>label_9</zorder>
  <zorder>editScientistSex</zorder>
  <zorder>editScientistYearOfBirth</zorder>
  <zorder>editScientistYearOfDeath</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
